setup.py

```diff
@@ -14,7 +14,7 @@
 
 from setuptools import setup
 
-version = '1.0.1'
+version = '1.1.0'
```

Comment:
I don’t see a reason why we should tie these two together. We should be able to iterate on pydatalab individually.